src/Icons/iconsMetaData.js

```diff
@@ -4,6 +4,13 @@ const VIEW = "View";
 export const iconsMetaData = [
   // DO NOT REMOVE THIS FOLLOWING LINE!
   // plop_marker:icon_metadata
+  {
+      name: "AssignedInSubItem",
+      file: ".svg",
+      description: "Describes a pulse that has a subitem that you are assigned to",
```

Comment:
open source, not related to pulse or sub-item, should describe the icon if possible